proprietary/basic-theme-design-tokens/src/brand/basic-theme/color.tokens.json

```diff
@@ -0,0 +1,41 @@
+{
+  "basic-theme": {
+    "color": {
+      "rood": {
+        "40": { "value": "#b1111c", "comment": "Button focus achtergrond" },
+        "50": { "value": "#d41422", "comment": "Standaard rood" },
+        "70": { "value": "#ec8383", "comment": "Used for focus outline" }
```

Comment:
Comments zijn soms in NL, soms in EN. Misschien handig om dit in 1 taal te houden?

Reply:
Nadat we de kleuren duidelijk hebben zal ik alle texten in een taal te hebben